Bottom: 3880c22443b939938a89ad668884e96b9ba0fd23
Top:    cbc44ef59750f921bd9632c98e8a6f3f26092205
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-20 23:22:00 +0100

hid/common: Draw the hole outline for pins / vias being moved or placed


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 14b6491..054610b 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -94,7 +94,7 @@ thindraw_moved_pv (PinType *pv, int x, int y)
   moved_pv.X += x;
   moved_pv.Y += y;
 
-  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, false, false);
+  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, true, false);
 }
 
 /* ---------------------------------------------------------------------------
@@ -573,7 +573,7 @@ DrawAttached (void)
         via.Mask = 0;
         via.Flags = NoFlags ();
 
-        gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &via, false, false);
+        gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &via, true, false);
 
         if (TEST_FLAG (SHOWDRCFLAG, PCB))
           {
